_scripts/lesson1-3.R

```diff
@@ -109,3 +109,33 @@
     
 }
 
+# APPLICATION ---
+
+{
+  
+  rm(list=ls()); # clean out the environment
+  options(show.error.locations = T); # give line numbers on errors
```

Comment:
Using T instead of TRUE can be problematic because, in R, you can declare a variable called T and assign it a value. Whereas, in R, TRUE is system defined and you cannot assign a value to it.

Try it in R:
T = FALSE;      # allowed!
TRUE = FALSE # not allowed